src/patient-registration/patient-registration.component.tsx

```diff
@@ -32,6 +34,7 @@ export interface FormValues {
   stateProvince: string;
   country: string;
   postalCode: string;
+  relationships: any[];
```

Comment:
Is there a better typing for `relationships: any[]`?

Reply:
Done....Introduced a type